srcpkgs/pdfarranger/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'pdfarranger'
+pkgname=pdfarranger
+version=1.7.1
+revision=1
+build_style=python3-module
+hostmakedepends="
```

Comment:
start listing the packages on this line, no need for a linebreak

Reply:
ok. I have deleted.